### ISSUE TITLE
forjj update failure

### ISSUE THREAD

**Issue by clarsonneur** [forj-oss/forjj#246]
When forjj is not executed for some times, the /tmp/forjj gets removed by the OS which then breaks `forjj update` as follow

```bash
2018/11/06 14:10:00 -------------------------------------------
2018/11/06 14:10:00 Running update on github...
2018/11/06 14:10:00 -------------------------------------------
2018/11/06 14:10:00 Forjj update issue. Failed to update 'github' source files. mkdir /tmp/forjj/forjj-210646907: no such file or directory
```

There is 2 possible workaround:
- recreate the `/tmp/forjj`
- change default value with `--plugins-socket-dirs-path` or env variable `PLUGINS_SOCKET_DIRS_PATH`

**Comment by clarsonneur** [comment]
The existing code, already do that. But it can't work in the context of Jenkins running forjj pipeline, because, Jenkins master is already started, but the host OS removed it. So the path disappeared.

So, the pipeline have to force a different socket path which won't be removed, but created properly.

So, to fix this, no change on forjj, probably, except the /tmp rule to authorize creating the full path when `/tmp/` is used.

**Comment by clarsonneur** [comment]
This issue will be fixed by #245 by updating help message which remind the usage of /run instead of /tmp for Jenkins pipeline context when Jenkins Master is started as DooD and forjj pipeline is configured to run from that Jenkins master instance.
